examples/kingworld/README.md

```diff
@@ -0,0 +1,15 @@
+# KingWorld with Bun runtime
+
+## Getting Started
+To get start with this template, simply paste the command in to your terminal:
```

Comment:
To get started with this template, simply paste this command in to your terminal: